tests/cli/RunCest.php

```diff
@@ -633,4 +633,209 @@ public function runBootstrapInSuiteConfig(\CliGuy $I)
+      )
+      ->addTest(HtmlReportRegexBuilder::newTestBuilder('CodeceptionIssue4413Cest', 'Case6b')
+        ->addMetaStep('Page\DemoPageObject: demo action1 with nested no metastep2')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+        ->addStep('no metaStep inside a private internal method')
+        ->addMetaStep('Page\DemoPageObject: demo action2')
+        ->addStep("I don't see file found", 'thisFileAgainDoesNotExist')
+      )
+      ->addTest(HtmlReportRegexBuilder::newTestBuilder('CodeceptionIssue4413Cest', 'Case7')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+      )
+      ->addTest(HtmlReportRegexBuilder::newTestBuilder('CodeceptionIssue4413Cest', 'Case8')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action2')
+        ->addStep("I don't see file found", 'thisFileAgainDoesNotExist')
+      )
+      ->addTest(HtmlReportRegexBuilder::newTestBuilder('CodeceptionIssue4413Cest', 'Case9')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action2')
+        ->addStep("I don't see file found", 'thisFileAgainDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+      )
+      ->addTest(HtmlReportRegexBuilder::newTestBuilder('CodeceptionIssue4413Cest', 'Case10')
+        ->addMetaStep('Page\DemoPageObject: demo action2')
+        ->addStep("I don't see file found", 'thisFileAgainDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+        ->addMetaStep('Page\DemoPageObject: demo action1')
+        ->addStep("I don't see file found", 'thisFileDoesNotExist')
+        ->addStep("I don't see file found", 'thisFileAlsoDoesNotExist')
+      );
```

Comment:
Getting error message that report doesn't match a monster pattern is not useful. 
This test is unmaintainable, because nobody will be able to fix it if it breaks.

It would be much better if each case was a separate test.

>  Fail  Contents of currently opened file does not match '/scenarioRow .*?CodeceptionIssue4413Cest .*? Case1_orig .*?scenarioRow .*?stepName .*?no metaStep.*?stepName .*?no metaStep.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case1 .*?scenarioRow .*?stepName .*?no metaStep.*?stepName .*?no metaStep.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case2 .*?scenarioRow .*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?no metaStep.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case3 .*?scenarioRow .*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case4 .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case5 .*?scenarioRow .*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action1 with nested no metastep.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?no metaStep inside a method.*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case5b .*?scenarioRow .*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action1 with nested no metastep2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?no metaStep inside a private internal method.*?stepName .*?no metaStep.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case6 .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action1 with nested no metastep.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?no metaStep inside a method.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case6b .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action1 with nested no metastep2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?no metaStep inside a private internal method.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case7 .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case8 .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case9 .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?scenarioRow .*?CodeceptionIssue4413Cest .*? Case10 .*?scenarioRow .*?stepName .*?Page\\DemoPageObject\: demo action2.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileAgainDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?stepName .*?Page\\DemoPageObject\: demo action1.*?substeps .*?stepName .*?I don't see file found.*?>&quot;thisFileDoesNotExist&quot;.*?stepName .*?I don't see file found.*?>&quot;thisFileAlsoDoesNotExist&quot;.*?/s'

Reply:
Thanks for the good hint: I'll try to break up the big test case (`RunCest::runHtmlCheckReport`) in individual test cases (one per `CodeceptionIssue4413Cest` test case)...

Reply:
DONE: Split up reproducer test case RunCest#runHtmlCheckReport into parameterized test based on `@dataProvider`;  try to improve output/report for `@dataProvider` based parameterization of reproducer test case RunCest#runHtmlCheckReport

=> Thus "only" the up-to-now unsupported 4 test cases of `CodeceptionIssue4413Cest` (Case7-10) are failing as expected (as aforementioned due to missing fix of https://github.com/Codeception/phpunit-wrapper/pull/62) in https://ci.appveyor.com/project/DavertMik/codeception/builds/27624814:
```
...
x RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:Case7) (0.15s)
The process cannot access the file because it is being used by another process.
x RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:Case8) (0.15s)
The process cannot access the file because it is being used by another process.
x RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:Case9) (0.15s)
The process cannot access the file because it is being used by another process.
x RunCest: Execute tests with html output and check html (CodeceptionIssue4413Cest:Case10) (0.15s)
...
```